test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsModelValidatorTest.cs

```diff
@@ -49,6 +51,80 @@ public static IEnumerable<object[]> Validate_SetsMemberName_OnValidationContext_
             }
         }
 
+        private const string LocalizationKey = "LocalizeIt";
+
+        public static IEnumerable<object[]> Validate_AttributesIncludeValues
```

Comment:
Use `TheoryData`. When we first started writing tests we used this pattern. We've since gone to the `TheoryData` pattern.